sql/core/src/test/scala/org/apache/spark/sql/JoinSuite.scala

```diff
@@ -191,6 +203,13 @@ class JoinSuite extends QueryTest with SharedSparkSession with AdaptiveSparkPlan
 //    ).foreach { case (query, joinClass) => assertJoin(query, joinClass) }
 //  }
 
+  test("broadcastable join with shuffle join hint") {
+    spark.sharedState.cacheManager.clearCache()
+    sql("CACHE TABLE testData")
```

Comment:
why do we need to cache?

Reply:
That's how we made sure to make the table broadcastable, it's also used in some other tests.